hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -128,6 +128,9 @@ switch ($OVSRunMode)
         }
     }
     "container" {
+        # In container mode, OVS services run in containers, not as host services.
+        # We can remove host services if they exist.
+        RemoveOVSService
```

Comment:
It is a bit tricky to remove OVS services when specifying that $OVSRunMode=container.